### PR TITLE
Fix multi lazy bug and add test

### PR DESCRIPTION
**Pull request by tufandevrim**
This PR fixes the issue with multi lazy config and outfile override issue.
Added a test for the use case, as well

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

File: index.js [hunk 1]
```diff
@@ -169,7 +171,13 @@ module.exports = function extractify(b, opts) {
                         id: lazyEntry,
                         order: 0
                     });
+
+                    if (lazyEntriesAll.indexOf(lazyEntry)>=0) {
```
Comment (juandopazo) [inline index.js]: Lint?

**Comment by juandopazo** [comment]
Thanks for fixing this! 👍

**Pull request merged**